_posts/cn/2018-12-29-apache-servicecomb-introduction-slides.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Apache ServiceComb开源微服务社区介绍"
+lang: cn
+ref: apache-servicecomb-introduction-slides
+permalink: /cn/docs/apache-servicecomb-introduction-slides/
+excerpt: "Apache ServiceComb开源微服务社区介绍PDF"
+last_modified_at: 2018-12-29T17:18:43+08:00
+author: Mabin
+tags: [微服务, PDF]
+redirect_from:
+  - /theme-setup/
+---
+
+
+
+
+**《Apache ServiceComb开源微服务社区介绍》**

```

Comment:
Please update the[ slide list](https://github.com/apache/servicecomb-website/blob/master/_pages/cn/about-slides.md)  for this slide.

Reply:
done,pls recheck,thks